src/cli/Peers.ts

```diff
@@ -0,0 +1,387 @@
+import fs from 'fs';
+import commander from 'commander';
+import { PolykeyAgent, PeerInfo } from '../lib/Polykey';
+import {} from '../../tmp/hole-punch-test/publicserver';
+import { actionRunner, pkLogger, PKMessageType, determineNodePath } from '.';
```

Comment:
Also import from `'.'`?

That's strange, generally the index should import from the constituents. Not the constituents import from index.

Can you make this module import from its deps and its deps should either be at the same level, or deeper.

And if we don't use aliases as discussed in the typescript-demo-lib, then we can minimize `../` imports by putting things nearer to the top-level and splitting up into more NPM packages.

Reply:
Also these functions seems like it should come a `utils` module. Something that exists at the `src/*` level like `src/utils.ts`. Would be ideal for alias but oh well.

Reply:
yeah sure, actually all of these are essentially utilities so I will put them in a `utils.ts` file :+1:

Reply:
well I guess these utility functions are only relevant for the CLI, things like resolving `nodePath` or `pkLogger`

Reply:
These are now in a utils file